Mathlib/Data/ENat/Lattice.lean

```diff
@@ -64,6 +64,9 @@ variable {f : ι → ℕ∞} {s : Set ℕ∞}
 lemma sSup_eq_zero : sSup s = 0 ↔ ∀ a ∈ s, a = 0 :=
   sSup_eq_bot
 
+lemma iSup_eq_zero : iSup f = 0 ↔ ∀ i, f i = 0 :=
```

Comment:
I suggest moving this lemma below `sSup_eq_zero'` to be consistent with the order of lemmas from `CompleteLattice.lean`.